src/contexts/account/useDefaultGroups.ts

```diff
@@ -0,0 +1,64 @@
+import { readContract } from '@wagmi/core';
+import { useCallback, useEffect, useState } from 'react';
+import { useBlockchain } from 'src/contexts/blockchain/useBlockchain';
+import { Option } from 'src/types';
+import { readFromCache, writeToCache } from 'src/utils/localSave';
+import { Address, useContractRead } from 'wagmi';
+
+const FEW_HOURS = 4 * 60 * 60 * 1000;
+
+const cacheKey = 'defaultGroups';
+
+export default function useDefaultGroups(): { activeGroups: Address[]; error: Option<Error> } {
+  const { defaultGroupStrategyContract } = useBlockchain();
+  const cachedGroups = readFromCache(cacheKey);
+  const [activeGroups, setActiveGroups] = useState<Address[]>([]);
+  const [error, setError] = useState<Error>();
+  const shouldRefetch = !(cachedGroups && cachedGroups.ts + FEW_HOURS > Date.now());
```

Comment:
if ts is the timestamp that they were last fetched then that would not ever be bigger than `Date.now()` 

`cachedGroups.ts + FEW_HOURS < Date.now()`